ui/app/AppLayouts/Chat/controls/MessageMouseArea.qml

```diff
@@ -38,7 +38,6 @@ MouseArea {
             }
             return;
         }
-        //TODO remove dynamic scoping
         if (mouse.button === Qt.LeftButton && isSticker && stickersLoaded) {
```

Comment:
`isSticker` and `stickerLoaded` not exists in this file. Where are they coming from now?

Reply:
@borismelnik, @PascalPrecht this is because the dynamic scoping chaos currently in chat section, I have mentioned it a lot of times :) simply how things were implemented before it was just having external to the file dependencies, this is where these properties are coming from, and since it was implemented like this now if anyone adds a property same name in the same file it won't work. This is an improvement that needs to be done in chat and in all sections, to remove the dynamic scoping because of exactly these inconveniences. I have left TODOs to mark that.